packages/vue_frontend_rpc/src/main.js

```diff
@@ -1,19 +1,57 @@
-import Vue from "vue";
-import App from "./App";
-
-import vuetify from "./plugins/vuetify";
+import { createApp, h } from "vue";
+import App from "./App.vue";
 import "material-design-icons-iconfont/dist/material-design-icons.css";
 import "./assets/css/globalStyles.css";
 import "@sap-devx/inquirer-gui/dist/form.css";
+import "@sap-devx/inquirer-gui-label-plugin/dist/labelPlugin.css";
 
 import Form from "@sap-devx/inquirer-gui";
+import "@sap-devx/inquirer-gui/dist/form.css";
+import vuetify from "./plugins/vuetify";
+import FileBrowserPlugin from "@sap-devx/inquirer-gui-file-browser-plugin";
+import FolderBrowserPlugin from "@sap-devx/inquirer-gui-folder-browser-plugin";
+import LabelPlugin from "@sap-devx/inquirer-gui-label-plugin";
+
+const plugins = [];
+
+const app = createApp({
+  render: () =>
+    h(App, {
+      ref: "appRef",
+    }),
+  data() {
+    return {};
+  },
+  methods: {
+    registerFormPlugins(formPlugins) {
+      console.log(formPlugins);
+      if (Array.isArray(formPlugins)) {
```

Comment:
why do you need the `isArray` check? you send always `plugins` which is initialized empty array, right?